src/options.html

```diff
@@ -134,21 +134,36 @@
             <br />
             <br />
 
+            <label>
+                <input type="checkbox" name="lineLengthLimitEnabled" />

```

Comment:
Another possible description could be `Show a line length limit ruler at the Nth column`, what do you prefer?